Modules/main.c

```diff
@@ -96,8 +96,10 @@ static const char usage_2[] = "\
          if stdin does not appear to be a terminal; also PYTHONINSPECT=x\n\
 -I     : isolate Python from the user's environment (implies -E and -s)\n\
 -m mod : run library module as a script (terminates option list)\n\
--O     : optimize generated bytecode slightly; also PYTHONOPTIMIZE=x\n\
--OO    : remove doc-strings in addition to the -O optimizations\n\
+-O     : remove assert and __debug__-dependent statements; add .opt1 before\n\
+         .pyc extension; also PYTHONOPTIMIZE=x\n\
+-OO    : do -O changes and also discard docstrings; add .opt2 before\n\
```

Comment:
Why do these say “.opt1” etc without a dash, but the other cases have a dash in “.opt-1” etc?

Reply:
I presume oversight on Cheryl's part.  I will fix.

Reply:
I presume oversight.  I checked my cache and '-' is correct.

Reply:
Yes, thank you for noticing that, Martin, and thank you, Terry, for fixing it.  It was an oversight.